turtlesim/tutorials/teleop_turtle_key.cpp

```diff
@@ -123,6 +184,43 @@ TeleopTurtle::TeleopTurtle():
   nh_->get_parameter("scale_linear", l_scale_);
 
   twist_pub_ = nh_->create_publisher<geometry_msgs::msg::Twist>("turtle1/cmd_vel", 1);
+  rotate_absolute_client_ = rclcpp_action::create_client<turtlesim::action::RotateAbsolute>(nh_, "turtle1/rotate_absolute");
+}
+
+void TeleopTurtle::sendGoal(double theta)
+{
+  auto goal = turtlesim::action::RotateAbsolute::Goal();
+  goal.theta = theta;
```

Comment:
@jacobperron https://ci.ros2.org/job/ci_windows/8159/warnings43Result/new/

> '=': conversion from 'double' to 'turtlesim::action::RotateAbsolute_Goal_>::_theta_type', possible loss of data

Reply:
See https://github.com/ros/ros_tutorials/pull/69 for a fix